ex5.py

```diff
@@ -0,0 +1,22 @@
+import numpy
+import numpy as np
+
+
+def add_vector_to_matrix(matrix: numpy.ndarray, vector:numpy.ndarray) -> numpy.ndarray:
+    """
+    This function adds a vector to each row of a given matrix.
+    :param matrix:
+    :param vector:
+    :return: the new matrix.
+    """
+    row_size, col_size = matrix.shape
+    result_matrix = np.empty_like(matrix)
+    for row in range(row_size):
+        result_matrix[row:] = matrix[row:] + vector
```

Comment:
The beauty of numpy is that in most cases for loops are not needed (and are usually less efficient), try using broadcasting.
https://numpy.org/doc/stable/user/basics.broadcasting.html